### DISCUSSION TITLE
Blackhole for Windows?

### DISCUSSION THREAD

**Discussion by ocmca** [ExistentialAudio/BlackHole#700] [General]
*Sorry, this has probably been the most asked question, but I can't find any posts with the few quick ones I did.*

### ***What would you awesome people recommend for me to loop back windows into Live, or OBS?***

I was able to get it working through our FocusRite Scarlet 2i2 coming from Ableton,
then the rear outputs into our little Harbinger mixer, 
where it splits, half going to the HS8s and 
the other half back into the FocusRite inputs where I was able to get OBS on the same box as Ableton to pick it up after much tinkering. But making audio loops (outside of Ableton) always makes me nervous, lol.

-----

Thanks for the recommendations. I know in my heart of hearts there's a software solution somewhere...

-Owen Alek

**Comment by devinroth** [comment]
Want to test a beta?

Join our discord channel for Windows and message me.

Devin

**Reply by ocmca** [reply]
For sure, I'll be there in a few.

**Comment by bigroundplanets** [comment]
I am having the same issue!
